### PR TITLE
feat: add Iowa liquor sales samples for Vertex AI Forecasting tutorial

### PR DESCRIPTION
**Pull request by thehardikv**
## Description

Adding samples of the Iowa liquor sales dataset to create a BQ Vertex AI Forecasting tutorial.

## Checklist

- [x] Please merge this PR for me once it is approved.
- [x] If this PR adds or edits a dataset or pipeline, it was reviewed and approved by the Google Cloud Public Datasets team beforehand.
- [x] If this PR adds or edits a dataset or pipeline, I put all my code inside  `datasets/<YOUR-DATASET>` and nothing outside of that directory.
- [x] If this PR adds or edits a dataset or pipeline that I'm responsible for maintaining, my GitHub username is in the [`CONTRIBUTORS` file](https://github.com/GoogleCloudPlatform/public-datasets-pipelines/blob/main/CONTRIBUTORS).
- [x] This PR is appropriately labeled.

### CODE REVIEW

File: datasets/iowa_liquor_sales/dataset.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    #
+    # Required Properties:
+    #   dataset_id
+    #
+    # Optional Properties:
+    #   friendly_name (A user-friendly name of the dataset)
+    #   description   (A user-friendly description of the dataset)
+    #   location      (The geographic location where the dataset should reside)
+    dataset_id: iowa_liquor_sales
+    description: "A time series dataset containing samples drawn from the Iowa Liquor Retail Sales dataset. Data were made available by the Iowa Department of Commerce. It is provided under the Creative Commons Zero v1.0 Universal license. For more details, see: https://pantheon.corp.google.com/marketplace/product/iowa-department-of-commerce/iowa-liquor-sales"
```
Comment (adlersantos) [inline datasets/iowa_liquor_sales/dataset.yaml:57]: Please use `console.cloud.google.com` over `pantheon.corp.google.com`

File: datasets/iowa_liquor_sales/dataset.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    # Google BigQuery dataset to namespace all tables managed by this folder
+    #
+    # Required Properties:
+    #   dataset_id
+    #
+    # Optional Properties:
+    #   friendly_name (A user-friendly name of the dataset)
+    #   description   (A user-friendly description of the dataset)
+    #   location      (The geographic location where the dataset should reside)
+    dataset_id: iowa_liquor_sales
```
Comment (adlersantos) [inline datasets/iowa_liquor_sales/dataset.yaml:56]: I think your target dataset is `iowa_liquor_sales_forecasting` (and not `iowa_liquor_sales`)? Let's provision that resource over this one.

**Review by adlersantos — CHANGES_REQUESTED** [review]
Just a few simple changes and we're good to go.

Please regenerate the Terraform and DAG files after the YAML revisions. Thanks!

File: datasets/iowa_liquor_sales/dataset.yaml (hunk 1 above)
Comment (thehardikv) [reply datasets/iowa_liquor_sales/dataset.yaml]: Done.

**Review by adlersantos — APPROVED** [review]

**Pull request merged**